pkg/runner/local_docker.go

```diff
@@ -231,23 +233,23 @@ func (r *LocalDockerRunner) Run(ctx context.Context, input *api.RunInput, ow *rp
 		// Start as many containers as group instances.
 		for i := 0; i < g.Instances; i++ {
 			// <outputs_dir>/<plan>/<run_id>/<group_id>/<instance_number>
-			odir := filepath.Join(r.outputsDir, input.TestPlan.Name, input.RunID, g.ID, strconv.Itoa(i))
+			odir := filepath.Join(r.outputsDir, input.TestPlan, input.RunID, g.ID, strconv.Itoa(i))
 			err = os.MkdirAll(odir, 0777)
 			if err != nil {
 				err = fmt.Errorf("failed to create outputs dir %s: %w", odir, err)
 				break
 			}
 
-			name := fmt.Sprintf("tg-%s-%s-%s-%s-%d", input.TestPlan.Name, testcase.Name, input.RunID, g.ID, i)
+			name := fmt.Sprintf("tg-%s-%s-%s-%s-%d", input.TestPlan, input.TestCase, input.RunID, g.ID, i)
```

Comment:
We need to filter characters docker doesn't allow. 

Right now if I start testground with this incantation
```
testground run single --builder docker:go --runner local:docker plans/placebo:ok -i 1

```

I get an error 
```
Apr 16 18:53:21.228605	WARN	engine run error: Error response from daemon: Invalid container name (tg-plans/placebo-ok-fa6c8935f543-single-0), only [a-zA-Z0-9][a-zA-Z0-9_.-] are allowed	{"req_id": "2a8d699c"}
```

Which is caused by the `/` character

Reply:
Thanks for reporting. The problem is that we were sending the client-side path of the plan to the server.

As of https://github.com/ipfs/testground/pull/849/commits/9c6aaffa9c88a3e4b62c3a29de9cbb86d7539164, we now replace it with the name of the plan as per the manifest.

We should still replace illegal characters for Docker/k8s, but that bug was already present before this PR. And it's not too bad to fail this way.